src/diffusers/models/normalization.py

```diff
@@ -43,8 +43,8 @@ def __init__(self, embedding_dim: int, num_embeddings: int):
 
     def forward(self, x: torch.Tensor, timestep: torch.Tensor) -> torch.Tensor:
         emb = self.linear(self.silu(self.emb(timestep)))
-        scale, shift = torch.chunk(emb, 2)
-        x = self.norm(x) * (1 + scale) + shift
+        scale, shift = torch.chunk(emb, 2, dim=1)
+        x = self.norm(x) * (1 + scale[:, None, :]) + shift[:, None, :]
```

Comment:
Can you please explain why this set of changes?

Reply:
The default argument for `dim` in `torch.chunk` is `dim=0` so it was splitting on the batch dimension. The second line change is to broadcast the scale and shift to the correct shape. This is exactly what is done elsewhere in the implementation that scale and shift are used. I could change it to `scale[:, None]` and `shift[:, None]` to match the other places it is used.

Reply:
Actually that caused some of the tests to fail. I will take a look at that.